pywiscat/wis1/report.py

```diff
@@ -141,5 +145,83 @@ def records_by_org(ctx, directory, file_list_file, group_by_authority, verbosity
         click.echo('No results')
 
 
+@click.command()
+@click.pass_context
+@cli_callbacks
+@click.option('--directory', '-d', required=False,
+              help='Directory with metadata files to process',
+              type=click.Path(resolve_path=True, file_okay=False))
+@click.option('--file-list', '-f', 'file_list_file',
+              type=click.Path(exists=True, resolve_path=True), required=False,
+              help='File containing JSON list with metadata files to process, alternative to "-d"')
+@click.option('--kpi', '-k', default=0, help='KPI to run, default is all')
+@click.option('--summary', '-s', is_flag=True, default=False,
+              help='Output only summary section of KPI test results for each file')
+@click.option('--brief-summary', '-S', 'brief_summary', is_flag=True, default=False,
+              help='Output extra brief summary of KPI test results for each file')
+@click.option('--group', '-g', 'group_by_authority', is_flag=True, default=False,
+              help='Group results by citation authority in the file identifier')
+def kpi(ctx, directory, file_list_file, group_by_authority, kpi, summary, brief_summary, verbosity):
+    """Runs pywcmp kpi tool on a each of the metadata files in a directory or list"""
```

Comment:
Runs kpi assessment on all metadata files in a directory or list

Reply:
This text is a bit too long, it gets truncated to "Runs kpi assessment on all metadata files in a...", therefore I have added a short help with this text "KPI assessment of metadata files".